beanquery/shell.py

```diff
@@ -157,24 +158,23 @@ def get_pager(self):
         if self.is_interactive:
             return pager.ConditionalPager(self.vars.get('pager', None),
                                           minlines=misc_utils.get_screen_height())
-        else:
-            file = (codecs.getwriter("utf-8")(sys.stdout.buffer)
-                    if hasattr(sys.stdout, 'buffer') else
-                    sys.stdout)
-            return pager.flush_only(file)
+        file = (codecs.getwriter("utf-8")(sys.stdout.buffer)
+                if hasattr(sys.stdout, 'buffer') else
+                sys.stdout)
+        return pager.flush_only(file)
 
-    def cmdloop(self):
+    def cmdloop(self, intro=None):
```

Comment:
W0221: Parameters differ from overridden 'cmdloop' method (arguments-differ)

Reply:
I think this pylint check is simply bogus as having a derived class having a method with a different signature is perfectly fine.